### PR TITLE
Adjust the porting doc

### PR DESCRIPTION
**Pull request by improbable-valentyn**
#### Description
Adjusted the porting doc to use ASCII quotes so the code blocks can be copy-pasted.
Mentioned the includes required for the `InitPlayerState` function.
#### Primary reviewers
@anne-edwards @m-samiec

### CODE REVIEW

File: docs/content/porting-unreal-project-to-gdk.md [hunk 1]
```diff
@@ -117,6 +117,14 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
         Super::InitPlayerState();
     }
     ```
+    
+1. In the file(s) you added the `InitPlayerState` function definition, add the following includes:
```
Comment (anne-edwards) [inline docs/content/porting-unreal-project-to-gdk.md]: - In the file(s) you added ...-> In the file(s) where you added...

- includes -> `#include`s

(the "includes" one looks weird I know - we can decide not to do it like this if you think it's too strange-looking)

Comment (anne-edwards) [reply docs/content/porting-unreal-project-to-gdk.md]: (these look like things I missed first time round!)

Comment (improbable-valentyn) [reply docs/content/porting-unreal-project-to-gdk.md]: Adjusted.

File: docs/content/porting-unreal-project-to-gdk.md [hunk 2]
```diff
@@ -117,17 +117,25 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
...
+    #include "EntityId.h"
+    #include "EntityRegistry.h"
+    ```
 
 ## Adding Unreal GDK configurations
 1. In `<GameRoot>\Config`, open `DefaultEngine.ini` and add:
 
     ``` ini
-    [\Script\Engine.Engine]
+    [/Script/Engine.Engine]
```
Comment (anne-edwards) [inline docs/content/porting-unreal-project-to-gdk.md]: Ah - did I go overboard with updating the forward slashes to back slashes? I thought I'd checked with Michael and only the command `-NetDriverOverrides=/Script/OnlineSubsystemUtils.IpNetDriver` needed forward slashes (but maybe I misunderstood!). So there might be some more rogue backslashes :(

Comment (improbable-valentyn) [reply docs/content/porting-unreal-project-to-gdk.md]: I went over the backslashes in this document specifically and they seem to be all good now.
We may need to check the other docs as well.
The forward slashes need to be used everywhere with `/Script/` (*not* `Scripts`, which is a folder)

Comment (anne-edwards) [reply docs/content/porting-unreal-project-to-gdk.md]: Got it, thanks! I'll add to our docs release checklist - check for rogue back slashes. Hopefully should be easy enough to do a quick search.

**Pull request merged**